config/log.ini

```diff
@@ -0,0 +1,11 @@
+[main]
+
+; level=DATA, PROTOCOL, DEBUG, INFO, NOTICE, WARN, ERROR, CRIT, ALERT, EMERG
```

Comment:
if the level is in lower case, the examples should be too.

Reply:
Agreed.

Reply:
done.